api/v1/verticadb_webhook.go

```diff
@@ -400,13 +400,46 @@ func (v *VerticaDB) getClusterSize() int {
 	return sizeSum
 }
 
-func (v *VerticaDB) hasValidDomainName(allErrs field.ErrorList) field.ErrorList {
+func (v *VerticaDB) hasValidSvcAndScName(allErrs field.ErrorList) field.ErrorList {
+	// check headless svc names
+	vdbName := v.ObjectMeta.Name
+	// vdb name will be used as is for headless svc name,
+	// thus match regex for service name (DNS-1035 label)
+	isValidHlSvcName := IsValidServiceName(vdbName)
+	if !isValidHlSvcName {
+		err := field.Invalid(field.NewPath("metadata").Child("name"),
+			v.ObjectMeta.Name,
+			fmt.Sprintf("vdb name is used as the headless service name, and hence must match regex '%s'",
```

Comment:
```suggestion
			fmt.Sprintf("vdb name must match regex '%s'",
```
Let's keep it simple

Reply:
will do